go.mod

```diff
@@ -2,30 +2,21 @@ module github.com/binance-chain/go-sdk
 
 require (
 	github.com/binance-chain/ledger-cosmos-go v0.9.9-binance.1
-	github.com/btcsuite/btcd v0.0.0-20181013004428-67e573d211ac
+	github.com/btcsuite/btcd v0.0.0-20190115013929-ed77733ec07d
```

Comment:
Shall we keep it consistent with cosmos repo? We use `v0.20.0-beta` in cosmos-sdk

Reply:
sure.